test/integration/helpers/Env.cs

```diff
@@ -15,25 +15,29 @@ public class Env
 
         private static void Init()
         {
+            if (_initialized) return;
+
             _env = new Dictionary<string, JsonElement>
             {
                 { "DEV", JsonDocument.Parse("true").RootElement }, 
                 { "isRemoteAppiumServer", JsonDocument.Parse("false").RootElement }, 
                 { "remoteAppiumServerUri", JsonDocument.Parse("\"http://localhost:4723\"").RootElement }
             };
 
-            if (_initialized) return;
-
             try
             {
-                _initialized = true;
-                var path = AppDomain.CurrentDomain.BaseDirectory;
-                var sr = new StreamReader(path + "env.json");
-                var jsonString = sr.ReadToEnd();
+                string path = AppDomain.CurrentDomain.BaseDirectory;
+                if (!path.EndsWith(Path.DirectorySeparatorChar.ToString()))
+                {
+                    path += Path.DirectorySeparatorChar;
+                }
+                StreamReader sr = new(path + "env.json");
+                string jsonString = sr.ReadToEnd();
                 _env = JsonSerializer.Deserialize<Dictionary<string, JsonElement>>(jsonString, new JsonSerializerOptions
                 {
                     PropertyNameCaseInsensitive = true
                 });
+                _initialized = true;
```

Comment:
is it expected is remains false if an exception happens?

Reply:
Yes, we don't want to set it to true if we fail.